README.md

```diff
@@ -121,15 +123,19 @@ smlbench/lookup-providers
 smlbench/store-get-value
 ```
 
-Then do
+Then run this comment to... (TODO: What does this do?)
```

Comment:
It runs the _lookup-peers_ test from the DHT plan, using the builder (setting up env + compilation) named docker:go (suggests that it compiles go inside docker) and runs it using the runner _local:docker_ (suggests that it is running in our local machine but locally).

Reply:
The command is indeed super verbose and definitely not easy to memorize. We haven't spent enough time on refining the user experience. It does need to happen though :)